modules/home-manager/gtk.nix

```
@@ -25,6 +25,11 @@ in
         default = [ "normal" ];
         description = "Catppuccin tweaks for gtk";
       };
+      gnome = mkOption {
+        type = lib.types.bool;
+        default = false;
+        description = "Wether to use the gtk theme as gnome shell's theme";
+      };
```

```suggestion
      gnomeShellTheme = mkEnableOption "the Catppuccin GTK theme for GNOME Shell"
```
this could also just be `shellTheme` maybe? i'll leave that up to you